multinode-demo/bootstrap-validator.sh

```diff
@@ -81,8 +81,9 @@ args+=(
   --ledger "$ledger_dir"
   --rpc-port 8899
   --snapshot-interval-slots 100
-  --identity-keypair "$identity_keypair"
-  --voting-keypair "$vote_keypair"
+  --identity "$identity"
+  --vote-account "$vote_account"
+  --authorized-voter "$vote_account"
```

Comment:
Should be `$identity` here?

Reply:
ah, thanks.  That was from my negative testing